src/main/java/Main.java.bak

```diff
@@ -19,7 +16,7 @@
 /**
  * This class represents the main class for the login and registration system.
  */
-public class Main {
+public class Mainjavabak {
```

Comment:
why change the class name? just rename the file itself

Reply:
I've renamed the class napo sir